src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -316,13 +316,14 @@ public Page<MeasureListDTO> getMeasures(
 
   public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
     if (StringUtils.isNotEmpty(cqlLibraryName)
-        && measureRepository.findByCqlLibraryName(cqlLibraryName).isPresent()) {
+        && !measureRepository.findAllByCqlLibraryName(cqlLibraryName).isEmpty()) {
```

Comment:
can we also check on active only? I think we missed this earlier iterations but we can address it now.